saleor/graphql/giftcard/tests/mutations/test_gift_card_add_note.py

```diff
@@ -0,0 +1,75 @@
+import graphene
+import pytest
+
+from .....giftcard import GiftCardEvents
+from .....giftcard.error_codes import GiftCardErrorCode
+from ....tests.utils import get_graphql_content
+
+GIFT_CARD_ADD_NOTE_MUTATION = """
+    mutation addNote($id: ID!, $message: String!) {
+        giftCardAddNote(id: $id, input: {message: $message}) {
+            errors {
+                field
+                message
+                code
+            }
+            giftCard {
+                id
+            }
+            event {
+                user {
+                    email
+                }
+                message
+            }
+        }
+    }
+"""
+
+
+def test_gift_card_add_note_as_staff_user(
```

Comment:
It will be nice to also add a test for the app.

Reply:
tests added.